repository_service_tuf_api/metadata.py

```diff
@@ -85,6 +85,79 @@ def post_metadata(payload: MetadataPostPayload) -> MetadataPostResponse:
     return MetadataPostResponse(data=data, message=message)
 
 
+class MetadataPutPayload(BaseModel):
+    roles: List[Roles.online_roles_values()]
```

Comment:
We need to change this as we would support custom target delegations' role names.
This should be any string.